sdk/tests/client/client_builder.rs

```diff
@@ -52,7 +52,9 @@ async fn client_builder() {
                 "vByteFactorData":1,
                 "vByteOffset":380
```

Comment:
This doesn't exist anymore?

Reply:
Right, I'll remove it in develop and will rebase 2.0 at some point.

Reply:
https://github.com/iotaledger/iota-sdk/pull/649